packages/calcite-components/src/components/combobox-item/combobox-item.e2e.ts

```diff
@@ -46,4 +47,43 @@ describe("calcite-combobox-item", () => {
   describe("disabled", () => {
     disabled("calcite-combobox-item", { focusTarget: "none" });
   });
+
+  it("should emit calciteInternalComboboxItemChange", async () => {
```

Comment:
To avoid testing internals, can we update existing `combobox`'s tests to cover the effect of changing these properties?